Jenkinsfile

```diff
@@ -42,14 +41,25 @@ pipeline {
         
             }
         }
+        
+        stage("Run oc-cluster-up"){
+            steps{
+                // TODO: Will be replaced with a step from pipeline-library which will be created later
+                build job: 'oc-cluster-up', parameters: [[$class: 'StringParameterValue', name: 'buildNode', value: "${env.NODE_NAME}"]], propagate: true, wait: true
```

Comment:
This [will be replaced](https://issues.jboss.org/browse/AEROGEAR-9314) by a function from external library